app/presenters/argo_show_presenter.rb

```diff
@@ -9,5 +9,15 @@ def heading
     CitationPresenter.new(@document).render
   end
 
-  attr_accessor :techmd, :cocina
+  def change_set
+    cocina.collection? ? CollectionChangeSet.new(cocina) : ItemChangeSet.new(cocina)
```

Comment:
Is the ArgoShowPresenter not used for APOs? Or might this be using the ItemChangeSet for APOs? https://app.honeybadger.io/projects/49894/faults/83811520

Reply:
Yeah, that seems like a possibility.